pages/en/blog/vulnerability/august-2023-security-releases.md

```diff
@@ -159,6 +159,6 @@ Releases will be available on, or shortly after, Tuesday August 8th 2023.
 
 ## Contact and future updates
 
-The current Node.js security policy can be found at https://nodejs.org/en/security/. Please follow the process outlined in https://github.com/nodejs/node/blob/master/SECURITY.md if you wish to report a vulnerability in Node.js.
+The current Node.js security policy can be found at https://nodejs.org/en/security/. Please follow the process outlined in https://github.com/nodejs/node/blob/main/SECURITY.md if you wish to report a vulnerability in Node.js.
```

Comment:
The canonical link should be https://github.com/nodejs/node/security/policy, not of any specific file or branch.

Reply:
Huh, didn't know about that GitHub-specific feature.

Reply:
→ #5995